flink-runtime-web/web-dashboard/src/app/app.component.less

```diff
@@ -137,9 +137,9 @@ nz-header {
 }
 
 nz-content {
+  display: flex;
```

Comment:
These changes are unrelated to the config object change. Given the size of the PR, can you please at least split this (and all other unrelated changes) into logical, unrelated commits? We can still do it in one PR.